data-gen/src/main/java/io/smartcat/data/loader/util/RandomizerImpl.java

```diff
@@ -0,0 +1,50 @@
+package io.smartcat.data.loader.util;
+
+import java.util.Random;
+
+/**
+ * Wrapper for java.util.Random.
+ */
+public class RandomizerImpl implements Randomizer {
```

Comment:
I added it because initially I wanted to create mock implementation of the Randomizer, to use it in tests. But it turned out, that it's not quite easy to make useful mock implementation. If we do not plan to create mocked implementations of Randomizer at all, that we can delete the interface.